src/js/edu-benefits/5490/config/form.js

```diff
@@ -320,6 +336,7 @@ const formConfig = {
               'ui:options': {
                 expandUnder: 'view:applicantServed'
               },
+              'ui:required': _.get('view:applicantServed'),
```

Comment:
I don't know how anyone feels about the clarity of this, but since lodash/fp auto-curries and undefined is falsy, it works here. Open to changing it.